massa-network-worker/src/node_worker.rs

```diff
@@ -376,8 +417,9 @@ impl NodeWorker {
                         },
                         Some(NodeCommand::SendOperations(operations)) => {
                             massa_trace!("node_worker.run_loop. send Message::SendOperations", {"node": self.node_id, "operations": operations});
-                            for chunk in operations.chunks(self.cfg.max_operations_per_message as usize) {
-                                if self.try_send_to_node(&writer_command_tx, ToSend::Msg(Message::Operations(chunk.into()))).is_err() {
+                            let ops: Vec<OperationId> = operations.into_iter().collect();
```

Comment:
I'm running into the below, think we would have to implement `From` for some of the prehashed structures, worth looking into in a follow-up if you think it's worth it...

```
error[E0277]: the trait bound `HashSet<OperationId, BuildHasherDefault<PreHashedMap<OperationId>>>: From<Chunk<'_, std::collections::hash_set::IntoIter<OperationId>>>` is not satisfied
   --> massa-network-worker/src/node_worker.rs:422:103
    |
422 | ...                   if self.try_send_to_node(&writer_command_tx, ToSend::Operations(chunk.into())).is_err() {
    |                                                                                             ^^^^ the trait `From<Chunk<'_, std::collections::hash_set::IntoIter<OperationId>>>` is not implemented for `HashSet<OperationId, BuildHasherDefault<PreHashedMap<OperationId>>>`
    |
    = help: the following implementations were found:
              <HashSet<T> as From<[T; N]>>
    = note: required because of the requirements on the impl of `Into<HashSet<OperationId, BuildHasherDefault<PreHashedMap<OperationId>>>>` for `Chunk<'_, std::collections::hash_set::IntoIter<OperationId>>`
```

Reply:
Filed https://github.com/massalabs/massa/issues/2623